libs/core/core/components/apps-switch.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import { Mode } from "Core/__deprecated__/common/enums/mode.enum"
+import ErrorApp from "Core/core/components/apps/error-app.component"
+import HelpApp from "Core/core/components/apps/help-app.component"
+import LicenseApp from "Core/core/components/apps/license-app.component"
+import TermsOfServiceApp from "Core/core/components/apps/terms-of-service-app.component"
+import PrivacyPolicyApp from "Core/core/components/apps/privacy-policy-app.component"
+import SarApp from "Core/core/components/apps/sar-app.component"
+import BaseAppContainer from "Core/core/components/apps/base-app/base-app-container.component"
+
+const AppsSwitch = () => {
```

Comment:
ensure that component is not re-rendered every 10 sec.

Reply:
https://github.com/mudita/mudita-center/assets/17147149/5a3fa921-6fdf-42a4-824b-8cec775436e4